plugins/source/mysql/client/list_tables.go

```diff
@@ -0,0 +1,75 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func Identifier(name string) string {
+	return fmt.Sprintf("`%s`", name)
```

Comment:
As it's heavily used during sync
```suggestion
	return "`" + name + "`"
```